pkgs/top-level/php-packages.nix

```diff
@@ -463,7 +463,7 @@ in {
           buildInputs = [ uwimap openssl pam pcre2 libkrb5 ];
           configureFlags = [ "--with-imap=${uwimap}" "--with-imap-ssl" "--with-kerberos" ];
           # Using version from PECL on new PHP versions.
-          enable = lib.versionOlder php.version "8.3";
```

Comment:
I agree that the other instances should use `meta.broken` but this one specifically is needed to be filtered out on PHP > 8.3 since the PHP 8.4 source tree stopped shipping it and this broken derivation would shadow the fallback definition above.

Reply:
Thanks for the insight. I’ve restored the logic using `lib.optionals` instead since it’s just this one package; let me know if you think that’s okay?

Reply:
Makes sense to me.